### PR TITLE
Rm reserved keyword char

### LINKED ISSUE #5778: Compression with Yui Compressor broken
## Version
at least from 3.0.0 and until 3.2.0

similar to #2934 

Compression broken by usage of reserverd "char" variable name  in `getHeightOfChar` here : https://github.com/fabricjs/fabric.js/blob/v3.0.0/dist/fabric.js#L25206

You can test it here : https://fr.piliapp.com/minify/yui-compressor/

### PR DESCRIPTION
**Pull request by pimpin**
Should fix #5778

### CODE REVIEW

**Comment by pimpin** [comment]
Notice I did not actualy build the dist file but manually fix it.
The minified file is not fixed. But both ot them should be fixed on next commited built.
And we have to backport fix into master branch.

**Comment by asturur** [comment]
Hi @pimpin please open a PR only on the master branch, without committing to the dist file at all.
I will release then a new version with the fix applied

File: dist/fabric.js [hunk 1]
```diff
@@ -25203,8 +25203,8 @@ fabric.Image.filters.BaseFilter.fromObject = function(object, callback) {
      * @param {Number} char the character number
      * @return {Number} fontSize of the character
      */
-    getHeightOfChar: function(line, char) {
-      return this.getValueOfPropertyAt(line, char, 'fontSize');
+    getHeightOfChar: function(line, _char) {
+      return this.getValueOfPropertyAt(line, _char, 'fontSize');
```
Comment (asturur) [inline dist/fabric.js]: should also fix the JSDOC 3 lines above

Comment (pimpin) [reply dist/fabric.js]: ok. LEts do that. Will you backport it and build all of 3.0.1 3.1.1 and 3.2.1 ?

**Comment by asturur** [comment]
i will put in 3.2.1 and if you really need i can do a one off backport and publish, but if ypir app works with 3.0 will work with 3.2.1 too

**Comment by pimpin** [comment]
Actually we are not sure it's working well with 3.0.0. We are upgrading from 1.5.0 and we did not complete all our checkup yet.
Currently I've patched the source file but we might bump to 3.2.1.

**Comment by asturur** [comment]
1.5 wow..... my first big release as contributor. Fabric changed a lot.

**Pull request closed without merging**